File: lab1/sesion-2/ejemplos_c/conceptos_c/estructuras/comparacion_lenguajes/java/Ejemplo2.java
public class Ejemplo2 {
    public static void main(String[] args) {
        Persona p1 = new Persona();

        // En Java, la asignación de strings es directa
        p1.nombre = "Ana";
        p1.edad = 22;

        System.out.println("Nombre: " + p1.nombre);
        System.out.println("Edad: " + p1.edad);
    }
}

File: lab1/sesion-2/ejemplos_c/conceptos_c/estructuras/comparacion_lenguajes/python/ejemplo2.py
class Persona:
    def __init__(self):
        self.nombre = ""
        self.edad = 0

if __name__ == "__main__":
    p1 = Persona()

    p1.nombre = "Ana"
    p1.edad = 22

    print(f"Nombre: {p1.nombre}")
    print(f"Edad: {p1.edad}")